web-common/src/features/canvas/inspector/chart/MarkSelector.svelte

```diff
@@ -99,14 +104,16 @@
   </div>
 {:else if selected === 0}
   <div class="pt-2">
-    <SingleColorSelector
-      small
-      theme={$theme}
-      markConfig={typeof markConfig === "string" ? markConfig : "primary"}
-      onChange={(newColor) => {
-        onChange(newColor);
-      }}
-    />
+    {#key `${isThemeModeDark}-${resolvedTheme.primary.hex()}-${resolvedTheme.secondary.hex()}`}
+      <SingleColorSelector
+        small
+        theme={resolvedTheme}
```

Comment:
Is the key necessary here? I removed the key block and played around with the color selector a bit. Seemed to work fine.